### PR TITLE
MC-42 Bind AddCard button to AuthorizeSwitch.

### PR DESCRIPTION
**Pull request by anyercastillo**
## Pictures
Screenshot_20210524_184304
Screenshot_20210524_184309

## Summary
Bind AddCard button with AuthorizeSwitch

## Ticket
https://spinbikes.atlassian.net/browse/MC-42

### CODE REVIEW

File: drop-in/src/main/java/com/adyen/checkout/dropin/ui/component/CardComponentDialogFragment.kt [hunk 1]
```diff
@@ -84,9 +86,25 @@ class CardComponentDialogFragment : BaseComponentDialogFragment() {
...
     override fun onChanged(paymentComponentState: PaymentComponentState<in PaymentMethodDetails>?) {
         // nothing, validation is already checked on focus change and button click
     }
+
+    private fun bindAddCardButtonWithAuthorizeSwitch() {
+        val switchCompat = dropInCardView
+            .cardView
+            .findViewById<SwitchCompat>(R.id.switch_storePaymentMethod)
+
+        // PM @Rana M said its important that the toggle is in the ON position by default
```
Comment (zhentan) [inline drop-in/src/main/java/com/adyen/checkout/dropin/ui/component/CardComponentDialogFragment.kt]: can we remove this comment? or at least rephrase it to say something like spec requires default to be true

Comment (anyercastillo) [reply drop-in/src/main/java/com/adyen/checkout/dropin/ui/component/CardComponentDialogFragment.kt]: definitely we should re-word it

Comment (anyercastillo) [reply drop-in/src/main/java/com/adyen/checkout/dropin/ui/component/CardComponentDialogFragment.kt]: Comment updated.

**Review by zhentan — APPROVED** [review]
I still don't really like the comments but I dont feel strongly enough about this to request/make change.
